### PR TITLE
Fix tensor reconstruction example

### PR DESCRIPTION
**Pull request by saurabheights**
## Type

-   [x] Bug fix (non-breaking change which fixes an issue): Fixes #
-   [ ] New feature (non-breaking change which adds functionality). Resolves #
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected) Resolves #

## Motivation and Context

The `integrate.py`  in `examples/python/t_reconstruction` did not work. I am using open3d v0.17.0 with python 3.10.10 on ubuntu 22.04.

1. ~~`store_true` no longer works, see [here](https://github.com/isl-org/Open3D/blob/master/examples/python/t_reconstruction_system/integrate.py#L86). ~~ MY BAD - This is not a bug.
2. Config file has `integration_mode` which can be removed in favor of `integrate_color` which is IMO easy to understand.
3. Documentation is no longer synced - http://www.open3d.org/docs/release/tutorial/t_reconstruction_system/integration.html#activation 
4. `integrate_color` is method argument [here](https://github.com/isl-org/Open3D/blob/master/examples/python/t_reconstruction_system/integrate.py#L25) but missing in [caller](https://github.com/isl-org/Open3D/blob/master/examples/python/t_reconstruction_system/integrate.py#L107)

## Checklist:

-   [x] I have run `python util/check_style.py --apply` to apply Open3D **code style**
    to my code.
-   [x] This PR changes Open3D behavior or adds new functionality.
    -   [x] Both C++ (Doxygen) and Python (Sphinx / Google style) **documentation** is
        updated accordingly.
    -   [ ] I have added or updated C++ and / or Python **unit tests** OR included **test
        results** (e.g. screenshots or numbers) here.
-   [x] I will follow up and update the code if CI fails.
    
-   [x] For fork PRs, I have selected **Allow edits from maintainers**.

## Description

Screenshot from 2023-06-13 23-06-34
Screenshot from 2023-06-13 23-06-02

- - -
This change is [Reviewable](https://reviewable.io/reviews/isl-org/Open3D/6207)

### CODE REVIEW

**Comment by saurabheights** [comment]
P.S.  I couldnt test changes for documentation if the new linking happens correctly. I need to do a full build locally for this but I am not sure how to do this. Can you provide me some direction?

**Review by theNded — APPROVED** [review]
LGTM, I will also work on improving it.

**Comment by theNded** [comment]
@ssheorey It is ready to merge, my follow-up upgrades will be in separate PRs.

File: examples/python/t_reconstruction_system/default_config.yml [hunk 1]
```diff
@@ -8,7 +8,6 @@ depth_folder: depth
 color_folder: color
 path_intrinsic: ''
 path_color_intrinsic: ''
-fragment_size: 100
```
[the remark this answers is not in the document]
Comment (saurabheights) [reply examples/python/t_reconstruction_system/default_config.yml:11]: @ssheorey Yes. The `fragment_size` key is given twice with same value 100 and there is no need of duplicate key, hence I removed it.

[the remark this answers is not in the document]
Comment (saurabheights) [reply examples/python/t_reconstruction_system/default_config.yml:11]: Also, I prefer to keep commits atomic, so reviewers can see reason of change - https://github.com/isl-org/Open3D/pull/6207/commits

**Comment by ssheorey** [comment]
Thanks @saurabheights !

**Pull request merged**